lib/mandrill_dm/message.rb

```diff
@@ -64,6 +64,17 @@ def html
       mail.html_part ? mail.html_part.body.decoded : mail.body.decoded
     end
 
+    def template
```

Comment:
It's `template_name`
https://mandrillapp.com/api/docs/messages.JSON.html#method=send-template